event-manager/src/main/java/evrentan/community/eventmanager/entity/EventVenue.java

```diff
@@ -51,7 +51,7 @@ public class EventVenue {
      *
      * @since 1.0.0
      */
-    @Column(name = "venue_id")
+    @Column(name = "venue_id", insertable = false, updatable = false)
```

Comment:
please check why we need insertable & updatable false. We should not need these.